stats/src/main/java/io/airlift/stats/DecayTDigest.java

```diff
@@ -127,11 +126,6 @@ public double valueAt(double quantile)
         return digest.valueAt(quantile);
     }
 
-    public List<Double> valuesAt(List<Double> quantiles)
```

Comment:
These methods serve two different purposes. Not always the caller has a fixed list of values, so a list is necessary.

Reply:
Is there a real use case where the caller has a variable list of quantiles ? I couldn't find any actual usage of this particular method.
For the same method on TDigest, I found all the existing usage of it be unnecessary and am proposing to removing it in https://github.com/trinodb/trino/pull/16811 
The implementation of this method is just converting the list to an array and then using the primitive version of the implementation. That can be done explicitly by the caller anyway.
If there is a real need for variable input, then there should an implementation that doesn't incur the overheads of List<Double> to double[] conversion. Otherwise it's just hiding the cost of conversions from the caller and can lead to callers not thinking about this unnecessary overhead.